modular_skyrat/modules/altborgs/code/robot_defines.dm

```diff
@@ -1,4 +1,6 @@
+/mob/living/silicon
+	var/datum/examine_panel/tgui = new() //create the datum
```

Comment:
Please rename the `tgui` variables to something that means something without seeing the definition, like `examine_panel` (that would work).